Stack/Opc.Ua.Core/Stack/Bindings/BufferManager.cs

```diff
@@ -117,14 +117,15 @@ public class BufferManager
         /// Constructs the buffer manager.
         /// </summary>
         /// <param name="name">The name.</param>
-        /// <param name="maxPoolSize">Max size of the pool.</param>
+        /// <param name="maxUsedQuota">Max allowed number of buffers to be used</param>
         /// <param name="maxBufferSize">Max size of the buffer.</param>
-        public BufferManager(string name, int maxPoolSize, int maxBufferSize)
+        public BufferManager(string name, int maxUsedQuota, int maxBufferSize)
         {
             int maxArrayLength = maxBufferSize + m_cookieLength;
             m_name = name;
             m_arrayPool = ArrayPool<byte>.Create(maxArrayLength, 4);
```

Comment:
needs some investigation if MemoryPool were a better choice here, it can dispose all buffers in use.